content/docs/06-integrations/00-kubernetes-generic.md

```diff
@@ -0,0 +1,624 @@
+---
+title: "Kubernetes"
+metaTitle: "Kubernetes"
+metaDescription: "Learn about the Kubernetes pack and how you can use it with your host clusters."
+hiddenFromNav: true
+type: "integration"
+category: ["kubernetes"]
+logoUrl: "https://registry.spectrocloud.com/v1/k8s-dashboard/blobs/sha256:2de5d88b2573af42d4cc269dff75744c4174ce47cbbeed5445e51a2edd8b7429?type=image/png"
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Kubernetes
+
+The Kubernetes pack supports several cloud and data center infrastructure providers. This pack defines the default properties we use to deploy Kubernetes clusters and enables most of the Kubernetes hardening standards that the Center for Internet Security (CIS) recommends.
```

Comment:
@ritawatson double check with Vish but we don't do any customization to the generic Kubernetes pack, so I don't think we can make CIS statements.

Reply:
Sent Vish a slack message to confirm.

Reply:
@karl-cardenas-coding Vish has confirmed this is correct and the second sentence re CIS is needed.